PERFORMANCE

```diff
@@ -37,6 +37,55 @@ To characterize Chapel performance, generally speaking...
   assignments are used to move chunks of data between locales.
 
 
+Experimental flags for improving performance
+--------------------------------------------
+
+The following compiler flags are intended to provide a preview of
```

Comment:
I wasn't sure what to suggest as an alternative to 'compiler flags', so I didn't suggest anything :) 

I think I like "config params" Anybody playing with these flags will probably know what that means

I was also thinking just "flags" might be ok